Makefile

```diff
@@ -1,6 +1,9 @@
 install:
 	obt install --verbose
 
+verify:
+	nbt verify --skip-layout-checks
+
 test:
 	obt build
-	obt verify
+	nbt verify --skip-layout-checks
```

Comment:
(shipping assets to prod not an issue for components so in this repo it doesn't matter)